sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -528,10 +530,10 @@ Note: The Healthcare Entities Analysis service is only available in the Standard
 - Linked Entity Recognition
 - Key Phrase Extraction
 - Sentiment Analysis
-- Custom Entity Recognition (see sample [here][recognize_custom_entities_sample])
-- Custom Single Label Classification (see sample [here][single_label_classify_sample])
-- Custom Multi Label Classification (see sample [here][multi_label_classify_sample])
-- Healthcare Entities Analysis (see sample [here][healthcare_action_sample])
```

Comment:
these link to the bespoke methods so removing from the "multiple analysis" section